src/jit/standalone/CMakeLists.txt

```diff
@@ -4,6 +4,10 @@ add_definitions(-DSELF_NO_HOST)
 add_definitions(-DFEATURE_READYTORUN_COMPILER)
 remove_definitions(-DFEATURE_MERGE_JIT_AND_ENGINE)
 
+if(CLR_CMAKE_TARGET_ARCH_I386 OR CLR_CMAKE_TARGET_ARCH_ARM)
```

Comment:
Since you've already posted this change here, we can simply take it as part of this PR. I didn't think that this PR was going to show up so soon :)